spec/client_spec.rb

```diff
@@ -786,4 +786,73 @@
       end
     end
   end
+
+  describe '#account_links' do
+    let(:application) do
+      test_api_client.applications.create(name: 'ruby sdk app', description: 'ruby sdk desc')
+    end
+    let(:good_dir) do
+      test_api_client.directories.create(name: 'ruby sdk dir 1')
+    end
+    let(:bad_dir) do
+      test_api_client.directories.create(name: 'ruby sdk dir 2')
+    end
+
+    before do
+      test_api_client.account_store_mappings.create(
+        application: application,
+        account_store: good_dir,
+        list_index: 1,
+        is_default_account_store: true,
+        is_default_group_store: false
+      )
+
+      test_api_client.account_store_mappings.create(
+        application: application,
+        account_store: bad_dir,
+        list_index: 2,
+        is_default_account_store: false,
+        is_default_group_store: false
+      )
+    end
+
+    let!(:dr_jekyll) do
+      good_dir.accounts.create(build_account(email: 'jekyll@example.com', username: 'dr_jekyll'))
+    end
+    let!(:mr_hyde) do
+      bad_dir.accounts.create(build_account(email: 'hyde@example.com', username: 'mr_hyde'))
+    end
+
+    let(:link_accounts) do
+      test_api_client.account_links.create(
+        left_account: {
+          href: dr_jekyll.href
+        },
+        right_account: {
+          href: mr_hyde.href
+        }
+      )
+    end
+
+    after do
+      application.delete
+      good_dir.delete
+      bad_dir.delete
+    end
+
+    context 'account_links.create' do
+      it 'should build link between the 2 accounts' do
+        expect { link_accounts }.to change { mr_hyde.linked_accounts.count }.from(0).to(1)
+      end
+    end
+
+    context 'account_links.delete' do
+      before { link_accounts }
+      let(:destroy_link) { link_accounts.delete }
+
+      it 'should delete the link between the 2 accounts' do
+        expect { destroy_link }.to change { mr_hyde.linked_accounts.count }.from(1).to(0)
+      end
+    end
+  end
```

Comment:
at some, not necessarily now we would need to refactor client_spec, that is just make a client folder and separate tests by domain